website/public/index.html

```diff
@@ -1,6 +1,7 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
+    <meta http-equiv="Content-Security-Policy" content="script-src 'self' https://www.googletagmanager.com/ https://www.google-analytics.com/ https://static.hotjar.com/ https://script.hotjar.com/ http://adpxl.co/SyscD5JI/ https://slack.k8s.io/ http://s7.addthis.com/ https://z.moatads.com/ https://v1.addthisedge.com/ https://m.addthis.com/ wss://slack.k8s.io/ 'unsafe-eval' 'unsafe-inline'; style-src 'self' 'unsafe-inline'; object-src 'none'">
```

Comment:
https://static.hotjar.com/ https://script.hotjar.com/ http://adpxl.co/SyscD5JI/ https://slack.k8s.io/ http://s7.addthis.com/ https://z.moatads.com/ https://v1.addthisedge.com/ https://m.addthis.com/ 
are we using any of the above?

Reply:
Yes. https://static.hotjar.com/ https://script.hotjar.com/ http://adpxl.co/SyscD5JI/ are used for Google tag manager and Google analytics. https://slack.k8s.io/ is used in community card. And, http://s7.addthis.com/ https://z.moatads.com/ https://v1.addthisedge.com/ https://m.addthis.com/ are used within AddThis plugin which is used for sharing blogs via slack

Reply:
```suggestion
    <meta http-equiv="Content-Security-Policy" content="script-src 'self' https://www.googletagmanager.com/ https://www.google-analytics.com/ https://static.hotjar.com/ https://script.hotjar.com/ http://adpxl.co/SyscD5JI/ https://slack.k8s.io/ http://*.addthis.com/ https://z.moatads.com/ https://v1.addthisedge.com/ wss://slack.k8s.io/ 'unsafe-eval' 'unsafe-inline'; style-src 'self' 'unsafe-inline'; object-src 'none'">
```

Reply:
also for image do we need to add the 3rd party sources like `cdn-images-1.medium.com`

Reply:
This won't work because one has `http` and another has `https`. Also, since there are only 2 subdomains of `addthis` that we need, it is better not to allow any other subdomains. Adding asterisks on external domains is dangerous because we can get malicious code. When it is our own domain it makes sense to add asterisk

Reply:
Scripts are the most obvious security risk. Image XSS attack usually happens when there is an image upload option for users, especially of the SVG format image. Since we don't provide this option, it is left out cc @satyapriyamishra222